services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -42,7 +40,7 @@ export default class CurrentSubscription extends React.Component {
       return this.editCreditCardElement();
     } else if (subscriptionStatus && subscriptionStatus.payment_method === 'Credit Card') {
       return <span>Credit Card</span>;
-    } else if (subscriptionType === 'School Sponsored' || subscriptionType === 'Trial') {
+    } else if (subscriptionType === 'Teacher Premium Trial') {
```

Comment:
Same with this and the rest of the places we're changing the display based on a plan name (and not just displaying it directly).